src/main/user-impl/java/com/mysql/cj/jdbc/ha/plugins/failover/FailoverConnectionPlugin.java

```diff
@@ -174,6 +174,7 @@ public class FailoverConnectionPlugin implements IConnectionPlugin {
 
   private long invokeStartTimeMs;
   private long failoverStartTimeMs;
+  boolean initialConnectionToReader;
```

Comment:
Also just noticed, should this field be `private`?

Reply:
It's used in a few tests, so I made it package private.